src/winter_onboarding_2021/fleet_management_service/views/layout.clj

```diff
@@ -1,13 +1,24 @@
 (ns winter-onboarding-2021.fleet-management-service.views.layout
   (:require [hiccup.page :refer [html5 include-css include-js]]))
 
-(defn application [request title & content]
+(defn application [request title content]
   (html5 [:head
```

Comment:
You can move the `html5` call to `wrap-layout`. This way the application can return a data structure and we can make better assertions than doing string matching.